includes/phing/build/main/subsite.xml

```diff
@@ -48,6 +48,9 @@
 
       <exec command="rsync -Lvr ${build.dist}/ ${build.subsite.dir}/" />
 
+      <!-- Copy drush configuration file. -->
+      <exec command="rsync -Lvr ${resources.dir}/drush/* ${build.platform.dir}/sites/default/" />
```

Comment:
You cannot use `${resources.dir}` , the files is inside the vendor/ec-europa/toolkit not in client resources. Also add a validation to only copy if file is not available in destination.